changes/4477-PrettyWood.md

```diff
@@ -0,0 +1 @@
+fix: always call __post_init__ when set in dataclasses
```

Comment:
```suggestion
fix: always call `__post_init__` when set in dataclasses
```

Reply:
it's what I wrote but somehow it has been added in bold

Reply:
it's markdown... this is the change file, not the PR title.

Reply:
I know. Just a VS code extension that messed up